### ISSUE TITLE
Marked text to selection

### ISSUE THREAD

**Issue by thsm-kb** [notepad-plus-plus/notepad-plus-plus#13430]
Feature request:

I would like to remove duplicate lines except empty lines. 
Approch: Use find to mark all the non-empty lines, and then do a 'Remove duplicate lines' on them. Result = nothing.

When lines are selected it is possible to do a Remove duplicate lines on the selection only. Not when they are marked.

So if I could 'select marked lines' it would solve my problem.

**Comment by dinkumoil** [comment]
> So if I could 'select marked lines' it would solve my problem.

No, it wouldn't. This is a two-fold issue. Not only that Notepad++ doesn't provide the ability to select lines marked as search results, it is also not able to do a "Remove duplicate lines" on multiple selections as this feature only works on a **block of selected lines**, i.e. a single selection.

**Comment by thsm-kb** [comment]
Thank you for clarifying the issue for me.

**Comment by notdodgeball** [comment]
The option to ignore empty lines when removing duplicates is being discussed here: #13173

About your first demand, I don't know an easy way to select marked or bookmarked lines, although we can do pretty much everything else: replace, copy, cut and paste onto them...
